geometry_msgs/Vector3 value
---
int32 success